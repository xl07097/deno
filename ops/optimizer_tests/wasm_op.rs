fn op_wasm(memory: Option<&mut [u8]>) {
  // @test-attr:wasm
}
