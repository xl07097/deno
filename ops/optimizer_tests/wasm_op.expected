=== Optimizer Dump ===
returns_result: false
has_ref_opstate: false
has_rc_opstate: false
has_fast_callback_option: false
needs_fast_callback_option: false
fast_result: Some(Void)
fast_parameters: [V8Value]
transforms: {0: Transform { kind: WasmMemory, index: 0 }}
is_async: false
fast_compatible: true
